lib/init.coffee

```diff
@@ -10,10 +10,13 @@ checkedAppend = (parameters, opt, args) ->
     parameters.push opt
     parameters.push(args...)
 
-makeParameters = (globals, ignore, file) ->
+makeParameters = (globals, ignore, std, file) ->
   parameters = ['-', '--no-color', '--codes', '--ranges', "--filename=" + file]
   checkedAppend parameters, '--globals', globals
   checkedAppend parameters, '--ignore', ignore
+  if std.length > 0
+    parameters.push '--std'
+    parameters.push(std.join '+')
```

Comment:
Should this keep the default list of standards, or replace as you are doing here?

If you wanted to enhance the default list instead of replacing, it should be something along the lines of this:
```coffeescript
parameters.push("+#{std.join('+')}")
```